### ISSUE TITLE
Test Failing 90% of the time

### ISSUE THREAD

**Issue by bpeters** [destinationdev/eth_enrollment#6]
https://github.com/destinationdev/eth_enrollment/blob/800357df35b6e1b0371a08e82d0b2bf081432099/test/enrollment.js#L333

Looks like this test fails 90% of the time and is the only one failing.

**Comment by bpeters** [comment]
```
93589776000000000000 'Original Balance'
95586696400000000000 'New Balance'
1996920399999991800 'Difference'
2000000000000000000 'initialUsdTuition * initialSpotRate'
3079600000000000 'gasUsed * gasPrice'
93589776000000000000 'New Balance - (initialUsdTuition * initialSpotRate) + (gasUsed * gasPrice)'
95586696400000000000 'Original Balance + (initialUsdTuition * initialSpotRate) - (gasUsed * gasPrice)'
```

```
93405724600000000000 'Original Balance'
95402645000000000000 'New Balance'
1996920399999991800 'Difference'
2000000000000000000 'initialUsdTuition * initialSpotRate'
3079600000000000 'gasUsed * gasPrice'
93405724599999990000 'New Balance - (initialUsdTuition * initialSpotRate) + (gasUsed * gasPrice)'
95402645000000010000 'Original Balance + (initialUsdTuition * initialSpotRate) - (gasUsed * gasPrice)'
```

First one passes, second does not. Looks like the reported gas is off by 10000.

**Comment by dmill** [comment]
Got it, yea I'll make this one a bit more fuzzy

**Comment by dmill** [comment]
addressed in commit: 2b64b97fc4c56cae5be6adbc9b9772fa26653ec6

I'm using `approximately` assertions in tests that require calculations based on gas used rather than `equals`

**Comment by dmill** [comment]
addressed in commit: 2b64b97fc4c56cae5be6adbc9b9772fa26653ec6

I'm using `approximately` assertions in tests that require calculations based on gas used rather than `equals`
